pkg/dpll/dpll.go

```diff
@@ -66,6 +92,12 @@ func (d *DpllConfig) CmdStop() {
 	glog.Infof("Ticker stopped %s", d.Name())
 	close(d.exitCh) // terminate loop
 	glog.Infof("Process %s terminated", d.Name())
+	if d.conn != nil {
+		d.nlCtxCancel()
```

Comment:
Maybe we need a waitgroup here, because we can close the connection before the goroutine will exit. This will lead to attempt to use a closed connection, error in logs